src/bootstrap/tarball.rs

```diff
@@ -206,9 +205,40 @@ impl<'a> Tarball<'a> {
                 .arg(input_tarballs)
                 .arg("--non-installed-overlay")
                 .arg(&this.overlay_dir);
```

Comment:
I think you can drop the two args above?

Reply:
Fixed!